selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/mavenplugin/CheckMavenPluginTest.java

```diff
@@ -48,97 +50,74 @@
   @Inject private Loader loader;
   @Inject private AskForValueDialog askDialog;
   @Inject private Git git;
-
-  @Inject
-  @Named("github.username")
-  private String gitHubUsername;
+  @Inject private TestProjectServiceClient testProjectServiceClient;
 
   @Inject private TestCommandServiceClient commandServiceClient;
 
   @BeforeClass
   public void setUp() throws Exception {
-    commandServiceClient.createCommand(
-        "cd /projects/" + PROJECT_NAME + " && git checkout contrib-12042015",
-        CHECKOUT_COMMAND,
-        TestCommandsConstants.CUSTOM,
-        workspace.getId());
-
+    URL resource = getClass().getResource("/projects/check-maven-plugin-test");
+    testProjectServiceClient.importProject(
+        workspace.getId(),
+        Paths.get(resource.toURI()),
+        PROJECT_NAME,
+        ProjectTemplates.MAVEN_SPRING);
     ide.open(workspace);
     projectExplorer.waitProjectExplorer();
   }
 
   @Test
-  public void mavenStatusBarShouldDisplayResolvingProjectMessage() {
```

Comment:
Why this and other tests were removed from the test class? Does it mean the corresponded functionality disappeared from the Che? Or those tests were useless and tested nothing?

Reply:
Status bar was removed from che6. But i hope it will be returned and we enabled this one again.

Reply:
I see. What about next test *shouldExecuteCommandAndWaitTextInConsole*? It checked message inside console.

Reply:
n this case it unnecessary step. Because it was done for checkout on tag in the just cloned project. After improving we don't need cloning project and checkout on tag it is extra steps that may be unstable.

Reply:
Also we check correct work of maven plugin. Cloning and checkout check in the Git package.

Reply:
I see. Thank you for the answer.